packages/driver/src/cy/commands/actions/type.ts

```diff
@@ -317,7 +324,25 @@ export default function (Commands, Cypress, cy, state, config) {
           }
         },
 
-        onEvent: updateTable || _.noop,
+        onEvent (id, key, event, value) {
+          if (updateTable) {
+            updateTable(id, key, event, value)
+          }
+
+          if (
+            // Firefox sends a click event when the Space key is pressed.
+            !Cypress.isBrowser('firefox') &&
```

Comment:
I think I'm misunderstanding, but If Firefox is the only browser that sends a click event with a space key, shouldn't this check be validating that the browser is Firefox? If this was chrome or Edge, this should send a click event that the browser wouldn't actually send giving the test a false-positive. 
```
Cypress.isBrowser('firefox')
```

Reply:
Unlike Chrome, Firefox automatically sends `click` event. Because of that, when we remove `!` here, Chrome does not send a `click` event and Firefox sends click events twice. 

This is the code to prevent this.

Reply:
I've added comment like below:

```js
// Firefox sends a click event when the Space key is pressed.
// We don't want send it twice.
!Cypress.isBrowser('firefox') &&
```

Reply:
Thanks for the explanation. I was concerned we were giving people a false-positive by sending a click event for chrome if it sent automatically.